.pre-commit-config.yaml

```diff
@@ -121,6 +121,8 @@ repos:
     args:
     - --install-types
     - --non-interactive
+    - --warn-unused-ignores
```

Comment:
Please, move these to a common config instead of duplicating. The existing args are only here because they differ across invocations.